frontend/ui/quickstart.lua

```diff
@@ -0,0 +1,70 @@
+--[[--This module is responsible for generating the quickstart guide.
+]]
+local DataStorage = require("datastorage")
+local FileConverter = require("apps/filemanager/filemanagerconverter")
+local gettext = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
+local T = require("ffi/util").template
+
+local quickstart_shown = G_reader_settings:readSetting("quickstart_shown") or nil
+local quickstart_shown_version = G_reader_settings:readSetting("quickstart_shown_version") or nil
```

Comment:
Looks like it's safe to assume that if `quickstart_show_version` is not `nil`, then `quickstart_shown` is `true`, i.e. `quickstart_shown` is redundant.